src/main.cpp

```diff
@@ -5935,14 +5935,14 @@ bool ProcessMessage(CNode* pfrom, std::string strCommand, CDataStream& vRecv, in
 
     else if (strCommand == NetMsgType::XPEDITEDBLK)
     {
-        // ignore the expedited message unless we are near the chain tip...
-        if (!fImporting && !fReindex && IsChainNearlySyncd())
+        // ignore the expedited message unless we are at the chain tip...
+        if (!fImporting && !fReindex && !IsInitialBlockDownload() && IsThinBlocksEnabled())
```

Comment:
The ThinBlocksEnabled check is now handle by HandleExpeditedBlock after ptschip's recent PR that went in.